Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * File:   FixedWing.c
+ * Author: Chris Hajduk
+ *
+ * Created on July 2, 2015, 8:04 PM
+ */
+
+#include "PWM.h"
+#include "AttitudeManager.h"
+#include "delay.h"
+#include "FixedWing.h"
+#include "ProgramStatus.h"
+
+#if VEHICLE_TYPE == FIXED_WING
+
+int outputSignal[NUM_CHANNELS];
+int control_Roll, control_Pitch, control_Yaw, control_Throttle;
+
+void initialization(){
+    setPWM(THROTTLE_OUT_CHANNEL, MIN_PWM);
+    
+    int channel = 0;
+    for (; channel < NUM_CHANNELS; channel++) {
```

Comment:
To keep the for loop style consistent, declare the channel above, and initialize it to 0 inside the for loop.